### ISSUE TITLE
idk how to run it, pls help

### ISSUE THREAD

**Issue by Devox628** [agausmann/perftree#16]
i have created my perft program in c++ but i cant get it to run,
i'm using a windows system and i have compiled the c++ file

```
F:\>cperftree F:\Documents\Cpp_C\chess_engine\main.exe
Error: Error { kind: NotFound, message: "program not found" }

F:\Documents\Cpp_C\chess_engine>perftree main.exe
Error: Error { kind: NotFound, message: "program not found" }
```

does it not accept compiled programs? _(the example on the readme shows a .sh file but i'm not on linux)_
i am sure that the path is not wrong

**Comment by kjhickman** [comment]
I think you may be missing the `stockfish` executable in your PATH.

**Comment by agausmann** [comment]
I agree, @Devox628 any luck with this? Are you able to run `stockfish` in the terminal? If not, it needs to be installed in PATH
